.gitignore

```diff
@@ -1,4 +1,3 @@
-.idea/
 .DS_Store
 __pycache__/
```

Comment:
why did you remove `*/.DS_Store` that used to be here?

More generally, why did you remove certain things and add certain things in .gitignore. The underlying reasoning is non-obvious to me as an observer

Reply:
I moved it down as there was a duplicate, but I can move it up if you want. I thought it would be better to be grouped together with relevant things

Several of them really should be added, for example `ENV` without which I get about 30K files suggested to be committed. Similarly, I removed `direnv`, `venv` and other common variations envs as they are typically causing complications.

Reply:
I don't think I deleted any of those tbh. I moved `.idea` down.  DS_store is also still there. (without *). 

>data/
tests/fasttext_data/
build (not build/)
cleanlab-docs
/docs/source/notebooks/spoken_digits

Reply:
> data/
tests/fasttext_data/
build (not build/)
cleanlab-docs
/docs/source/notebooks/spoken_digits

looking at the git history, I don't think they were ever added to this repo

Reply:
Doh I think I was comparing against the wrong .gitignore file, please ignore the specific list here. 

The general principle is just to clarify what things are being added/deleted in your PR, and when these are based on personal preferences VS based on better code organization.